orphedomos-maven-plugin/pom.xml.releaseBackup

```diff
@@ -0,0 +1,97 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
S: we should delete this file - these backups are just used the by the release plugin in case it needs to rollback.  But we can safely delete them.